client/lib/plans/constants.js

```diff
@@ -752,6 +842,130 @@ export const FEATURES_LIST = {
 			'Custom polls, surveys, ratings, and quizzes for the ultimate in customer and reader engagement.'
 		)
 	},
+
+	[ FEATURE_CORE_JETPACK ]: {
+		getSlug: () => FEATURE_CORE_JETPACK,
+		getTitle: () => i18n.translate( 'Core Jetpack Services' ),
+		getDescription: () => i18n.translate(
+			'Stats, themes, and promotion tools.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Stats, customization, and promotion tools' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).